### DISCUSSION TITLE
加上-e https://music.163.com参数后无法使用

### DISCUSSION THREAD

**Discussion by wukibaka** [1715173329/UnblockNeteaseMusic#465] [Q&A]
如题

**Comment by 1715173329** [comment]
图，配置，日志，版本，一个没有。
建议出门找个算命先生。

**Reply by wukibaka** [reply]
算命先生解决不了啊

**Reply by wukibaka** [reply]
版本与日志
![Screenshot_20220116-121335__01.jpg](https://user-images.githubusercontent.com/51240031/149647059-f3b823b4-81d8-49d3-84aa-48571ab528e2.jpg)

**Reply by wukibaka** [reply]
客户端为安卓7.2.22

**Comment by pan93412** [comment]
看起來替換成功了。試試看重新啟動應用？

**Reply by wukibaka** [reply]
服务器无报错，但客户端报无法缓冲歌曲，不加-e https://music.163.com参数正常
![Screenshot_20220117-073904.jpg](https://user-images.githubusercontent.com/51240031/149682869-4fc0baae-c265-46af-aa45-33cf06fe8c59.jpg)

![Screenshot_20220117-073636__01.jpg](https://user-images.githubusercontent.com/51240031/149682884-59c677ca-c02f-4482-b00b-a779f0b90cec.jpg)

**Reply by 1715173329** [reply]
不加能用就不加好了，没说非得加上
